envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -3,6 +3,16 @@ stack_env: example_ci-ceph_swift
 
 state_path_base: /opt/stack/data
 
+swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
+swift_undercloud_floating_ip: "{{ swift_floating_ip }}"
+swift_fqdn: "swift.{{ fqdn }}"
+
+etc_hosts:
+  - name: "{{ fqdn }}"
+    ip: "{{ undercloud_floating_ip }}"
+  - name: "{{ swift_fqdn }}"
+    ip: "{{ swift_undercloud_floating_ip }}"
```

Comment:
Logging is enabled for that nightly log and we do `test/run deploy --extra-vars "@${WORKSPACE}/logging_extra_vars.yml"` which will overwrite your etc_host with the etc_host we define for logging.  This mean that anything you define in etc_host that's not in whatever the logging stuff defines, will be lost.  I'll work on a patch to use lineinfile for logging to eliminate this problem.